src/sentry/static/sentry/app/views/onboarding/createProject.jsx

```diff
@@ -125,20 +147,39 @@ const CreateProject = createReactClass({
         {accessTeams.length ? (
           <OnboardingProject {...stepProps} />
         ) : (
-          <div>
-            <h4>
-              {t(
-                'You cannot create a new project because there are no teams to assign it to.'
-              )}
-            </h4>
-            <Link to={`/organizations/${slug}/teams/new/`} className="btn btn-primary">
-              {t('Create a Team')}
-            </Link>
-          </div>
+          <Panel
+            title={t('Cannot Create Project')}
+            body={
```

Comment:
I don't think this is too complicated where we should refactor it, especially when it's not being reused. Simple JSX (that isn't reused) should be kept inline imo because it's easier to read and you don't have to search around for the render method.